test/acceptance/cli-test.js

```diff
@@ -430,6 +430,18 @@ describe('ember-template-lint executable', function() {
       });
     });
   });
+
+  describe('editorconfig', function() {
+    it('should exit without error and any console output', function() {
+      let result = run(['app/templates/application.hbs'], {
+        cwd: './test/fixtures/with-editorconfig',
+      });
+
+      expect(result.exitCode).toEqual(0, 'exits without error');
+      expect(result.stdout).toBeFalsy();
+      expect(result.stderr).toBeFalsy();
+    });
+  });
```

Comment:
Would you mind leveraging the (newish) fixturify setup instead of adding new fixture files?

Reply:
Yes, should I extend `without-errors` fixture with `.editorconfig`?

Reply:
Sorry for the confusion, I’m saying that we should just not use fixtures at all. I’ll drum up an example for you from the recent work in 2.0.

Reply:
https://github.com/ember-template-lint/ember-template-lint/blob/4279eb30f9c7c20954951d26b74e1b34dcdb5b89/test/unit/get-config-test.js#L22

Reply:
Thanks, I got it and will update later.